modules/assistant.js

```diff
@@ -104,12 +104,12 @@ MixIn(PrefCustomization, SimpleCustomization.prototype);
 
 
 function MultipleCustomization(aParams) {
-  this.customizations = [new PrefCustomization(p) for (p of aParams)];
+  this.customizations = !!aParams ? aParams.map(p => new PrefCustomization(p)) : [];
```

Comment:
This looks like it might have fixed a bug? Should the same pattern be used other places or, or was this a special case?